app/views/records/edit_fields/_provenance.html.erb

```diff
@@ -1,4 +1,15 @@
 <% unless controller_name == 'collections' %> 
-  <%= f.input :provenance, as: :multi_value_with_help, input_html: { rows: '14', type: 'textarea'}, required: false %>
+  <% if current_user.admin? %>
+    <%= f.input :provenance, as: :multi_value_with_help, input_html: { rows: '14', type: 'textarea'}, required: false %>
+  <% else %>
+    <% unless @generic_file.provenance.blank? %>
```

Comment:
Should properly handle the provenance field of the "Additional fields" in file upload. Modifications should occur in either BatchEditForm or BatchController.